### DISCUSSION TITLE
Notatka do zamówienia - odczyt/zapis

### DISCUSSION THREAD

**Discussion by nemonno** [allegro/allegro-api#4742] [Suggestions - resolved]
Zrobiłem własną listę wysyłkową, jednak niestety nie można poprzez API pobrać i wyświetlić "Notatki do zamówienia". Taka opcja jest tylko przez WWW.
Sugeruję zatem dodanie możliwości odczytu notatki do zamówienia przez API oraz ewentualnie także możliwości jej dodania przez API.

**Comment by JamesFordV3** [comment]
podbijam suestię, bardzo przydałaby się taka opcja

**Comment by demonic80** [comment]
Witam,
czy coś wiadomo w temacie?

**Reply by AureliuszBrussy** [reply]
Nie wdrożyliśmy jeszcze tej sugestii. Jeżeli coś się zmieni - będziemy to komunikować oraz damy znać w wątku.

**Comment by Lukasz-Zurek** [comment]
Udostępniliśmy dzisiaj notatkę sprzedawcy w zamówieniach. Więcej informacji w naszym [newsie](https://github.com/allegro/allegro-api/issues/7409).
